doc/releases/release-notes-4.4.rst

```diff
@@ -173,6 +156,12 @@ New APIs and options
 
   * :c:macro:`COND_CASE_1`
 
+* Video
+
+  * :kconfig:option:`CONFIG_VIDEO_BUFFER_POOL_HEAP_SIZE`
+  * :kconfig:option:`CONFIG_VIDEO_BUFFER_POOL_ZEPHYR_REGION`
+  * :kconfig:option:`CONFIG_VIDEO_BUFFER_POOL_ZEPHYR_REGION_NAME`
```

Comment:
It is tempting to shorten them as:
- `CONFIG_VIDEO_BUFFER_POOL_SIZE` (or `_BUFFER_HEAP_SIZE`?)
- `CONFIG_VIDEO_BUFFER_REGION` (or `_BUFFER_MEMORY_REGION`?)
- `CONFIG_VIDEO_BUFFER_REGION_NAME`